doc/release-notes/whats-new.en.rst

```diff
@@ -31,6 +31,8 @@ total of <z> contributors have participated in this development cycle.
 New Features
 ------------
 
+- traffic_ctl has a new command ``monitor`` do show a continuously updating list of
```

Comment:
```suggestion
- traffic_ctl has a new command ``monitor`` to show a continuously updating list of
```

Reply:
This hasn't changed. Is the original version correct?

Reply:
I tried to use GitHub's suggestion feature but it seems it didn't actually do the change.  I'll try manually.